systest/tests/common.go

```diff
@@ -292,15 +292,20 @@ func (c *txClient) nonce(ctx context.Context) (uint64, error) {
 	return getNonce(ctx, c.node, c.account.Address)
 }
 
-func (c *txClient) submit(ctx context.Context, tx []byte) (*txRequest, error) {
-	txid, err := submitTransacition(ctx, tx, c.node)
-	if err != nil {
-		return nil, fmt.Errorf("submit to node %s: %w", c.node.Name, err)
+func (c *txClient) submit(ctx context.Context, tx []byte, tries int) (*txRequest, error) {
```

Comment:
tries should not be a required parameter to submit function. it either should be an option or removed from submit completely, and set when txClient is initialized

Reply:
used a default value for number of attempts. 

i feel eventually may need to wrap retries around the gRPC and rest/k8s calls. it's frustrating to fail the test immediately when those fail.